### ISSUE TITLE
Does the fastavro dependency version need to be pinned?

### ISSUE THREAD

**Issue by deleted** [ynqa/pandavro#39]
`setup.py` has:
```python
    install_requires=[
        # fixed versions.
        'fastavro==1.5.1',
        'pandas>=1.1',
        # https://pandas.pydata.org/pandas-docs/version/1.1/getting_started/install.html#dependencies
        'numpy>=1.15.4',
    ],
```

This causes a dependency resolution failure for me because I'm using another package that requires `fastavro>=1.5.4`.

Would it be possible to relax that requirement to be `'fastavro>=1.5.1`

**Comment by deleted** [comment]
I submitted a fix in #42

**Comment by jbvaningen** [comment]
I would like to +1 this. 

Relaxing this pinned requirement would also be helpful in making `pandavro` compatible with Python 3.11 and beyond, because `fastavro==1.5.1` only supports up to Python 3.10 ([see docs](https://pypi.org/project/fastavro/1.5.1/))
